app/src/main/java/com/naxtlevelofandroiddevelopment/marvelgallery/presentation/character/CharacterProfileActivity.kt

```diff
@@ -0,0 +1,56 @@
+package com.naxtlevelofandroiddevelopment.marvelgallery.presentation.character
+
+import android.content.Context
+import android.os.Bundle
+import android.view.MenuItem
+import com.naxtlevelofandroiddevelopment.marvelgallery.R
+import com.naxtlevelofandroiddevelopment.marvelgallery.model.MarvelCharacter
+import com.naxtlevelofandroiddevelopment.marvelgallery.presentation.common.*
+import kotlinx.android.synthetic.main.activity_character_profile.*
+
+class CharacterProfileActivity : PresenterBaseActivity(), CharacterProfileView {
```

Comment:
usually Activity being the view in MVP sets Presenter in onCreate(). I would not use word "Presenter" for your BaseActivity. View delegates to presenter but the two should be separate.